### PR TITLE
options.logoMaskable

### PR DESCRIPTION
**Pull request by carlhopf**
options.logoMaskable support (string, array)

files are loaded via webpack alongside default logo, and then injected to favicons as options.manifestMaskable

based on my other pull request, which enables array support for options.logo/logoMaskable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,6 +118,8 @@ plugins: [
   new FaviconsWebpackPlugin({
     // Your source logo (required)
     logo: './src/logo.png',
+    // Your maskable source logo (optional)
+    logo: './src/logo-maskable.png',
```
Comment (andy128k) [inline README.md:122]: shouldn't this be `logoMaskable: ...`?

**Comment by carlhopf** [comment]
amended the commit to correct typo in readme, plus a fix that was failing tests

File: src/index.js [hunk 2]
```diff
@@ -560,6 +615,18 @@ function mergeManifests(manifest1, manifest2) {
   return mergedManifest;
 }
 
+function findAndMap(filePaths, fileContents) {
+  return filePaths.map((filePath) => {
+    const fileContent = fileContents.filter(
+      (fileContent) => fileContent.filePath === filePath
+    )[0];
```
Comment (andy128k) [inline src/index.js:622]: It seems like [`find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) fits better here.

**Comment by carlhopf** [comment]
rebased and cleaned up, moved to array filter+find

**Comment by carlhopf** [comment]
rebased and cleaned up, added type and annotations

File: src/options.d.ts [hunk 3]
```diff
@@ -1,6 +1,8 @@
 export interface FaviconWebpackPlugionOptions {
   /** our source logo - can be png or svg (required) */
   logo: string | string[],
+  /** our maskable source logo - can be png or svg (optional) */
+  logoMaskable: string | string[],
```
Comment (andy128k) [inline src/options.d.ts:5]: As soon as it is optional, it should be reflected in the type too.

**Review by andy128k — CHANGES_REQUESTED** [review]

File: src/options.d.ts (hunk 3 above)
Comment (carlhopf) [reply src/options.d.ts]: done 👍

**Comment by andy128k** [comment]
Thanks!

**Pull request merged**